### DISCUSSION TITLE
Doesn't connect anymore after ~100 connected users

### DISCUSSION THREAD

**Discussion by PayaMousavi** [angristan/openvpn-install#1395] [Help]
Hello,
there isn't any limit on server config but when around 100 users get connected new users can't
i think problem is for local ip pool because tun0 is /24 subnet
any help or suggestion?

**Comment by randshell** [comment]
Can you try "Solution 1" from [here](https://www.wikitechy.com/tutorials/linux/how-to-increase-the-maximum-number-of-tcp-ip-connections-in-linux)?

> tun0 is /24 subnet

You can connect 253 clients with this setting

**Comment by HenryNe** [comment]
Run `journalctl -f` on the server while a new client can not connect.
There should be an error or any messages for incomming call.

Lock in your syslog. There should find such line:
`MULTI: TCP INIT maxclients=1024 maxevents=1028
`
That says, 1024 clients maximal.

On /24 net with option "topology subnet" can connect 253 clients, exactly from 10.8.0.2 to 10.8.0.254. [check ipcalc](http://jodies.de/ipcalc?host=10.8.0.0&mask1=24&mask2=)

Don't confuse with the old and default "topology net30", there it was 64 clients from 10.8.0.5 to 10.8.0.253 with a step of 4.

**Comment by MistFenix** [comment]
Please anybody can help me with multiple clients? I can't add on my server.conf
